src/index.html

```diff
@@ -2,17 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
```

Comment:
Use 2 spaces for indentation in your file. The indentation here is inconsistent, with some lines using more than 2 spaces. Ensure that each level of nesting is offset by 2 spaces from its parent.